discord/commands/core.py

```diff
@@ -155,16 +155,14 @@ def unwrap_function(function: Callable[..., Any]) -> Callable[..., Any]:
 
 def _validate_names(obj):
     validate_chat_input_name(obj.name)
-    if obj.name_localizations:
-        for locale, string in obj.name_localizations.items():
-            validate_chat_input_name(string, locale=locale)
+    for locale, string in obj.name_localizations.items():
+        validate_chat_input_name(string, locale=locale)
 
 
 def _validate_descriptions(obj):
     validate_chat_input_description(obj.description)
-    if obj.description_localizations:
-        for locale, string in obj.description_localizations.items():
-            validate_chat_input_description(string, locale=locale)
+    for locale, string in obj.description_localizations.items():
+        validate_chat_input_description(string, locale=locale)
```

Comment:
this suggestion vanished into the abyss for some reason, here it is again
```suggestion
    if obj.description_localizations:
        for locale, string in obj.description_localizations.items():
            validate_chat_input_description(string, locale=locale)
```